recipes/libpfm4/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conan import ConanFile
+from conan.tools.gnu import Autotools, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+from conan.tools.files import get, rmdir, copy, replace_in_file
+import os
+
+
+class Libpfm4Conan(ConanFile):
+    name = "libpfm4"
+    license = "MIT"
+    homepage = "https://github.com/wcohen/libpfm4"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = ("A helper library to program the performance monitoring events")
+    topics = ("perf", "pmu", "benchmark", "microbenchmark")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            try:
+                del self.options.fPIC
+            except Exception:
+                pass
+        try:
+            del self.settings.compiler.cppstd
+        except Exception:
+            pass
+        try:
+            del self.settings.compiler.libcxx
+        except Exception:
+            pass
+
+    def layout(self):
+        basic_layout(self)
+
+    def source(self):
+        get(self, **self.conan_data['sources'][self.version],
+                  strip_root=True,
+                  destination=self.source_folder)
```

Comment:
```suggestion
        get(self, **self.conan_data["sources"][self.version], strip_root=True, destination=self.source_folder)
```
There is a bug in hooks, it will faster adapting your recipe than fixing it.